rules/otbn.bzl

```diff
@@ -148,6 +148,45 @@ def _otbn_binary(ctx):
         ),
     ]
 
+def _otbn_test(ctx):
+    """This rule is for standalone OTBN unit tests, which are run on the host
+    via the OTBN simulator.
+
+    It first generates binaries using the same method as otbn_binary, then runs
+    them on the simulator. Tests are expected to count failures in the w0
+    register; the test checks that w0=0 to determine if the test passed.
+    """
+    providers = _otbn_binary(ctx)
+
+    # Extract the output .elf file from the output group.
+    elf = providers[1].elf.to_list()[0]
+
+    # Create a simple script that runs the OTBN simulator on the .elf file and
+    # checks if the w0 register is 0.
+    simulator_cmd = "{} {} --dump-regs -".format(ctx.file._simulator.path, elf.short_path)
+    expected_string = "0x0000000000000000000000000000000000000000000000000000000000000000"
```

Comment:
Do you know if Starlark supports the python string multiplication operator? (I.e., `'0' * 32`).

Reply:
Yes, it does, fixed!